src/components/btc-anchor-card.tsx

```diff
@@ -23,7 +23,15 @@ export const BtcAnchorBlockCard: React.FC<FlexProps & { block: Block }> = ({ blo
               label: {
                 children: 'Bitcoin block height',
               },
-              children: `#${block.burn_block_height}`,
+              children: (
+                <Link
+                  as="a"
+                  target="_blank"
+                  href={`https://www.blockchain.com/btc/block/${block.burn_block_height}?utm_source=stacks_explorer`}
```

Comment:
Included the `utm_source` param here as in the other links, but I am not sure exactly how it's used by the blockchain.com explorer. I noticed that the param is not included in some other links (added here: https://github.com/hirosystems/explorer/pull/811). Any thoughts on whether `utm_source` should be added to the `<BtcStxBlockLinks />` component?

Reply:
I think we should not add `utm_source` here which is commonly used for marketing campaigns. We are not responsible for blockchain.com marketing and it might even mess up their analytics. They can always know the source by using the `window.document.referrer`